src/Microsoft.Data.Analysis/IntDataFrameColumn.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data.Analysis
+{
+    public partial class IntDataFrameColumn : PrimitiveDataFrameColumn<int>
+    {
+        public IntDataFrameColumn(string name, IEnumerable<int?> values) : base(name, values) { }
+
+        public IntDataFrameColumn(string name, IEnumerable<int> values) : base(name, values) { }
+
+        public IntDataFrameColumn(string name, long length = 0) : base(name, length) { }
+
+        public IntDataFrameColumn(string name, ReadOnlyMemory<byte> buffer, ReadOnlyMemory<byte> nullBitMap, int length = 0, int nullCount = 0) : base(name, buffer, nullBitMap, length, nullCount) { }
+
+        internal IntDataFrameColumn(PrimitiveDataFrameColumn<int> intColumn) : base(intColumn.Name, intColumn._columnContainer) { }
```

Comment:
I don't see this ctor being used. Can we remove it until it is necessary?

Reply:
We would also not need to make `_columnContainer` internal on the PrimitiveDataFrameColumn class.

---
In reply to: [387168118](https://github.com/dotnet/corefxlab/pull/2857#discussion_r387168118) [](ancestors = 387168118)

Reply:
Yup. Removed in all the new columns